set-quotas.py

```diff
@@ -0,0 +1,191 @@
+#   Copyright 2016 Massachusetts Open Cloud
+#
+#   Licensed under the Apache License, Version 2.0 (the "License"); you may
+#   not use this file except in compliance with the License. You may obtain
+#   a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#   WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#   License for the specific language governing permissions and limitations
+#   under the License.
+"""Update a project's OpenStack quotas using Google Sheets data
+
+For each entry in the Google Sheet, the script will:
+    - update the specified quotas
+    - generate an email describing the update
+    - send the email to the requestor, cc'ing all project users and anyone
+      else specified in the config file cc_list parameter
+
+Usage:
+    python set-quotas.py
+"""
+import ConfigParser
+from keystoneclient.v3 import client
+from keystoneauth1.identity import v3
+from keystoneauth1 import session
+
+from quotas import QuotaManager
+from message import TemplateMessage
+import spreadsheet
+
+
+class ProjectNotFoundError(Exception):
+    """The specified project does not exist"""
```

Comment:
I am not sure how well would this work. As in current addusers script, we might have to manually edit the form, as there is no proper way to check for the syntaxes/validate the names in the match_keystone_project or parse_rows functions.

And are the entries in the google from manually entered by the user? If that's the case we might want to include some validation for the entries.